docs/topics/api/reviewers.rst

```diff
@@ -311,16 +318,42 @@ Please note, that once a review is submitted the drafted comments are being clea
 
 .. http:get:: /api/v4/reviewers/addon/(int:addon_id)/versions/(int:version_id)/draft_comments/
 
-    Retrieve an exising draft.
+    Retrieve existing draft comments for a specific version.
```

Comment:
Can we make this section look more like the other listing docs -- that is, at least specify `results` as an array of comment objects? Example:

<img width="706" alt="Screenshot 2019-07-10 10 05 38" src="https://user-images.githubusercontent.com/55398/60980651-7d58b500-a2fa-11e9-966a-ae1011b62213.png">

The docs, as is, look more like a detail response where one object is returned.

Reply:
Absolutely! To be honest, I seems to have looked at a wrong endpoint when I got myself "inspired" for the docs, I'll update and change the GET docs to interoperate pagination more properly.

Reply:
Fixed all that in f9909aa957.

![Screenshot_2019-07-10 Reviewers — addons-server 3 0 documentation](https://user-images.githubusercontent.com/139033/60987959-f67bfa80-a342-11e9-85d9-56f485c495c7.png)

Does that look better?

Reply:
Yes, this looks good. Thanks.